kotlin/Readme.md

```
@@ -22,12 +22,18 @@
        - Gradle task is registered to generate jar by running the command `./gradlew :injivcrenderer:build` which creates jar in the `build/libs` folder
        - Run Tests using `./gradlew testDebugUnitTest` or `./gradlew testReleaseUnitTest` based on the build type.
 
+### Runtime Dependencies
+- The library does not bundle its runtime dependencies in the generated aar or jar.
+- The runtime dependencies are to be explicitly added in the consumer application to avoid runtime failures.
+
 ### API
-- `generateCredentialDisplayContent(credentialFormat: CredentialFormat, wellknownJsonString: String? = null, vcJsonString: String)` - expects the Verifiable Credential, Well-known Json and Credential Format as input and returns the list of replaced SVG Templates.
+- `generateCredentialDisplayContent(credentialFormat: CredentialFormat, wellknownJsonString: String? = null, vcJsonString: String, qrCodeData: String? = null)` - expects the Verifiable Credential, Well-known Json, Credential Format and QR Code Data as input and returns the list of replaced SVG Templates.
     - `credentialFormat` - Enum to specify the credential format. Currently only LDP_VC format is supported.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)